test/cpp/end2end/server_builder_plugin_test.cc

```diff
@@ -232,8 +232,9 @@ class ServerBuilderPluginTest : public ::testing::TestWithParam<bool> {
   void RunCQ() {
     void* tag;
     bool ok;
-    while (cq_->Next(&tag, &ok))
+    while (cq_->Next(&tag, &ok)) {
       ;
```

Comment:
No need for this anymore, now that there are braces around it.

Reply:
Done